### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by slightlytyler**
Fixed a typo (react-formal-bootstrap => react-bootstrap) and improved the code snippet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,11 @@ input from `react-formal-bootstrap/lib`, to take advantage of the value parsing
...
 ```js
 import React from 'react';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
 import Form from 'react-formal';
 import * as yup from 'yup';
 
-import ControlLabel from 'react-bootstrap/lib/ControlLabel';
-
-import FormGroup from 'react-formal-bootstrap/lib/FormGroup';
```
Comment (jquense) [inline README.md:26]: this is actually a component of react-formal-bootstrap! :)

**Comment by slightlytyler** [comment]
Weird cus it was throwing an error when I copied the import statement from
the snippet, lemme double check it

**Comment by slightlytyler** [comment]
@jquense I see no file at the path `react-formal-bootstrap/lib/FormGroup`. Assuming this is a bug then?

**Comment by slightlytyler** [comment]
Oh it just looks the changes from January were never released

**Comment by jquense** [comment]
Oops. I should do that.

**Comment by jquense** [comment]
released!

**Comment by slightlytyler** [comment]
@jquense awesome! much thanks 🍾

**Pull request closed without merging**